src/views/Stage.vue

```diff
@@ -231,60 +225,67 @@ export default class Map extends Vue {
   }
 
   private nextTurn(animation: string) {
-    this.animation = animation
-    // Vue.nextTick(() => {
-    this.loopTurn()
-    // })
-    // this.generateGrid()
-
-    this.placeEntities()
-    // this.calculateObserveRange()
+    this.animation[0] = animation
```

Comment:
Hoe is die extendable, soos sal ons maklik animations kan add vir elke entity so? Verstaan net nie helemaal code nie

Reply:
ja dit kan, het dit net vir die player vir nou gedoen